backend/monitoring/dashboard.py

```
@@ -1404,11 +1405,27 @@ async def get_available_test_suites():
+                    
+                    # Test if the issue is with the WHERE clause or column names
+                    simple_result = await conn.fetch("SELECT suite_name, enabled FROM test_suite_configurations LIMIT 3")
+                    logger.info(f"Sample data: {simple_result}")
+                    
+                except Exception as debug_error:
+                    logger.error(f"Debug query also failed: {debug_error}")
+                
                 return StandardResponse(
                     status="success", 
-                    message="Test configurations unavailable - database schema needs to be updated",
+                    message=f"Test configurations unavailable - database schema needs to be updated",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)